ApiContract/response.json5

```diff
@@ -0,0 +1,45 @@
+//Response data
+{
```

Comment:
If post request was for advertisement and all the properties related to advertisement are at root level then , in response, advertisement related properties should be at root level.